actions-sms.php

```diff
@@ -432,226 +417,221 @@ function delstandnote($number, $standName, $message)
 
 function standNote($number, $standName, $message)
 {
-
-    global $db;
     $userId = getUser($number);
 
-
-    $result=R::getAll("SELECT standId FROM stands where standName=:standName", [':standName' => $standName]);
-    if ($result->num_rows!=1) {
+    $result = R::getAll("SELECT standId FROM stands where standName=:standName", [':standName' => $standName]);
+    if (count($result) != 1) {
         sendSMS($number, _("Stand")." ".$standName._("does not exist").".");
         return;
     }
 
-    $row =$result->fetch_assoc();
-    $standId=$row["standId"];
-
-    $result=R::getAll("SELECT userName from users where number=:number", [':number' => $number]);
-    $row =$result->fetch_assoc();
-    $reportedBy=$row["userName"];
+    $row = $result[0];
+    $standId = $row["standId"];
 
+    $result = R::getAll("SELECT userName from users where number=:number", [':number' => $number]);
+    $row = $result[0];
+    $reportedBy = $row["userName"];
```

Comment:
yes, this seems to be the same.
see file common.php and these functions that should be used all over:
`function getprivileges($userid)
{
    $user=R::load('users', $userid);
    if ($user->id) {
        return $user->privileges;
    }
    return false;
}

function getusername($userid)
{
    $user=R::load('users', $userid);
    if ($user->id) {
        return $user->username;
    }
    return false;
}

function getphonenumber($userid)
{
    $user=R::load('users', $userid);
    if ($user->id) {
        return $user->number;
    }
    return false;
}

function getuserid($number)
{
    $user=R::findOne('users', 'number=?', [$number]);
    if (!empty($user)) {
        return $user->id;
    }
    return false;
}`